src/EventStore.Core.Tests/Services/PersistentSubscription/OutstandingMessageCacheTests.cs

```diff
@@ -54,7 +54,7 @@ public void can_remove_duplicate()
             cache.StartMessage(new OutstandingMessage(id, null, Helper.BuildFakeEvent(id, "type", "name", 1), 0), DateTime.Now);
             cache.Remove(id);
             Assert.AreEqual(0, cache.Count);
-            Assert.AreEqual(long.MinValue, cache.GetLowestPosition());
+            Assert.AreEqual(long.MaxValue, cache.GetLowestPosition());
```

Comment:
This seems like an unrelated change. Possibly due to a bad rebase. This looks like the changes from a recent PR that was merged (https://github.com/EventStore/EventStore/pull/1302).